department-of-veterans-affairs/abd-vro
# Helm: add triggering-actor to container metadata

## What was the problem?

Sometimes, it's unclear who deployed certain containers, for [example](https://dsva.slack.com/archives/C04QLHM9LR0/p1692031390786019), which makes it difficult to diagnose and fix failed or undeployable containers.

Knowing who deployed certain containers, we can ask that person what the intended state should be in order to fix the problem.

Associated tickets or Slack threads:
- #1921

## How does this fix it?


Add `triggering-actor` to container metadata.

## How to test this PR
[Deploy a container](https://github.com/department-of-veterans-affairs/abd-vro-internal/actions/runs/5896264593) and check it's metadata/annotations:
<img width="610" alt="image" src="https://github.com/department-of-veterans-affairs/abd-vro/assets/55255674/f45e9fe0-47e9-4e43-b725-49ded915c54d">

Note: `vro/triggering-actor` (and `vro/deploy-time` if deploying `latest` image)
